registration/forms.py

```diff
@@ -43,7 +43,7 @@ class BetaTestApplicationForm(NgModelFormMixin, NgFormValidationMixin, NgModelFo
     """
     class Meta:
         model = BetaTestApplication
-        exclude = ('user', 'status')
+        exclude = ('user', 'status', 'instance')
```

Comment:
@smarnach Yeah, having to rebase can sometimes be a bit annoying for reasons like that. What I like to do to be safe is merge master into my branch, tag the tip of the branch with a new branch name (to create an easily-accessible "backup" of the branch state), then rebase onto master. At that point, I can diff my rebased branch with the pre-rebase tip; they should be 100% identical, and if they're not, I know a problem happened in the rebase.

That's a bit more work, but is a safer pattern to use when rebases are being required IMHO.